dev-tools/validation/check-adr-review-status.py

```
@@ -0,0 +1,181 @@
+                f"{path.relative_to(root)}: completed human review is not backed "
+                "by a checked review-queue entry"
+            )
+
+        if status == "Proposed - Pending Human Review":
+            if fields.get("Human review status") != "Pending":
+                fail(
+                    f"{path.relative_to(root)}: pending proposal requires "
+                    "'Human review status: Pending'"
+                )
+        elif status == "Accepted" and number not in PRE_FORMAL_REVIEW_RECORDS:
+            if not linked_reviews:
```

Ruff (0.15.21)

[warning] Use a single `if` statement instead of nested `if` statements

(SIM102)